hazelcast/src/main/java/com/hazelcast/monitor/LocalMapStats.java

```diff
@@ -210,4 +212,20 @@
      * @return statistics object for the Near Cache
      */
     NearCacheStats getNearCacheStats();
+
+    /**
+     * Returns the number of queries performed on the map.
+     */
+    long getQueryCount();
```

Comment:
can we have a javaDoc here to explain the difference of this method from `getIndexedQueryCount`? When there is no index what this method and `getIndexedQueryCount` return?

Reply:
fixed